src/ol/interaction/PolygonBrush.js

```diff
@@ -210,6 +242,17 @@ class PolygonBrush extends Draw {
   getBrushRadius() {
     return this.sketchPointRadius_;
   }
+
+  abortDrawing_() {
```

Comment:
Don't override this function. Instead, call `super()` and only delete the new invisible sketchPoint here (if `sketchFeature` is `true`.